steps/plandek-deployment/1.0.0/step.yml

```diff
@@ -0,0 +1,61 @@
+title: Plandek - deployment
+summary: |
+  Enables you to track and visualize build and deployment events within Plandek's metrics and dashboards.
+description: |
+  Wrapper implementing the Plandek /deployment POST API. Used to complete a deployment within Plandek for a given release cycle.
+  More details can be found here: https://api.plandek.com/deployments/v1/ui/
+website: https://github.com/secretescapes/bitrise-step-plandek-deployment
+source_code_url: https://github.com/secretescapes/bitrise-step-plandek-deployment
+support_url: https://github.com/secretescapes/bitrise-step-plandek-deployment/issues
+published_at: 2023-08-17T20:33:43.844566+02:00
+source:
+  git: https://github.com/secretescapes/bitrise-step-plandek-deployment.git
+  commit: 36162daa4df25ae888d66964fe47eb43f91395e8
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_always_run: true
+is_skippable: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      Plandek API token.
+    is_required: true
+    is_sensitive: true
```

Comment:
Please add `is_expand: true`, as it is required in case is_sensitive is true.

Reply:
@lpusok fyi: from what I understand `is_expand: true` is not needed as it's default value when `is_sensitive: true` -> https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#sensitive-inputs but I don't mind leaving it as is.

Reply:
You are right, it can be omitted.